cli/src/commands/upload.ts

```diff
@@ -75,22 +79,36 @@ export default class Upload extends BaseCommand {
         });
 
         let skipUpload = false;
-        if (!options.skipHash) {
+
+        let skipAsset = false;
+        let existingAssetId: string | undefined = undefined;
+
+        if (!options.skipHash && !options.album) {
```

Comment:
Not uploaded assets can now be added to albums without issues. I think it can probably be simplified to 
```suggestion
        if (!options.skipHash) {
```

Reply:
Done!